### ISSUE TITLE
Consider data release for historic pulls & quantile calcs

### ISSUE THREAD

**Issue by lindsayplatt** [USGS-VIZLAB/gw-conditions#30]
Might be worth making this a data release & include the decisions for how we got some GW equivalent data despite different parameter codes.

**Comment by collnell** [comment]
Agree - can we use this as an opportunity for you to walk me through the data release process?

**Comment by lindsayplatt** [comment]
Yes, am very experienced now with 1 under my belt 🤪 But I do know the general process now even if I have to ask others for some hints & details!
